content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -213,6 +213,14 @@ which detects if traffic redirection is set up correctly, and blocks the pod sta
 The CNI DaemonSet will detect and evict any pod stuck in such state. When the new pod starts up, it should have traffic redirection set up properly.
 This mitigation is enabled by default and can be turned off by setting `values.cni.repair.enabled` to false.
 
+This repair capability can be further configured with different RBAC permissions to help mitigate the theoretical attack vector detailed in [`ISTIO-SECURITY-2023-005`](/news/security/istio-security-2023-005/) .  By setting the below fields to true/false as required, you can select the Kubernetes RBAC permissions granted to the Istio CNI.
+
+|Configuration                    | Roles       | Notes
```

Comment:
Maybe we should have another column on how it works for each mode:

delete: pods are deleted, and when rescheduled they will have the correct configuration
label: pods are only labeled. This mode requires a user to detect when pods with these labels are set and take manual action (typically, deleting the pod)
repair: pods are dynamically reconfigured to have appropriate configuration. When the container restarts, the pod will continue normal execution.

Reply:
@pmerrison Thoughts on John's comment.